profile.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz App</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header class="header">
+      <h1 class="header__headline">Quiz App</h1>
+    </header>
```

Comment:
Your header could also be a component of its own. Therefore, your could move the stylings for this part in a file named `header.css` that is inside your other components folder. :)